bugbug/utils.py

```diff
@@ -19,6 +19,20 @@
 TASKCLUSTER_DEFAULT_URL = "https://taskcluster.net"
 
 
+def compress(path):
+    cctx = zstandard.ZstdCompressor()
+    with open(path, "rb") as input_f:
+        with open(f"{path}.zst", "wb") as output_f:
+            cctx.copy_stream(input_f, output_f)
+
+
+        
+def decompress(path):
+    dctx = zstandard.ZstdDecompressor()
+    with open(path, "rb") as input_f:
```

Comment:
You're reading from the file which has to be extracted and writing to the file from which we should extract, I guess you got this backwards :)

I suggest testing the functions yourself manually with a Python script (write a file, compress it, remove the original file, decompress the archive, check that the output file contains the same content you wrote at the beginning).